main.go

```diff
@@ -154,14 +155,18 @@ func main() {
 			log.Fatalln("tls.LoadX509KeyPair:", err)
 		}
 		clientCert = append(clientCert, certs)
-		mysql.RegisterTLSConfig("custom", &tls.Config{
+		const customTLSName string = "customTLS"
```

Comment:
I think you can drop the `string`. The `custom` sounds better than `customTLS` to me. :P